Dockerfile

```
@@ -0,0 +1,24 @@
+# --- Stage 1: Build with Rust --- (amd64)
+FROM rustlang/rust:nightly-slim@sha256:e1fb83983ca45a2f2afe386077cdf6873a7c0a496a5ae6e469b25dc1a90b1561 AS builder
```

- We should not be building with nightly Rust. Why did you select it?
- Where does `rustlang/rust` pull from?

https://hub.docker.com/_/rust suggests we should be using:
```suggestion
FROM rust:1-slim@sha256:TODO AS builder
```